moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -1422,6 +1422,9 @@
               <property name="text">
                <string>Filter</string>
```

Comment:
Is _Filter_ the right text description for the button?

Reply:
I can't see how it would fit. I changed it.